git-branchless-lib/src/core/check_out.rs

```diff
@@ -9,6 +9,7 @@ use cursive::utils::markup::StyledString;
 use tracing::instrument;
 
 use crate::git::{CategorizedReferenceName, GitRunInfo, MaybeZeroOid, Repo, WorkingCopySnapshot};
+use crate::util::ExitCode;
```

Comment:
Check out [proc_exit](https://docs.rs/proc-exit/).  I ran into a similar situation and created it to help with cases like this

Reply:
@epage `proc_exit::Code` isn't `#[must_use]` ☹️